src/general/GameWorld.cs

```diff
@@ -113,6 +126,8 @@ public GameWorld()
 
         // Register glucose reduction
         TimedEffects.RegisterEffect("reduce_glucose", new GlucoseReductionEffect(this));
+
+        UnlockProgress.World = this;
```

Comment:
This should not be required, if this is then UnlockProgress object is not correctly being loaded from saves...